gradle.properties

```diff
@@ -1,2 +1,2 @@
-version = 2.16-SNAPSHOT
+version = 2.17-SNAPSHOT
```

Comment:
Is this intended? 2.16 is the next version to be released

Reply:
done